### DISCUSSION TITLE
Show `host` in left prompt elements without terminal icon

### DISCUSSION THREAD

**Discussion by binaryfire** [romkatv/powerlevel10k#2794] [General]
Hi all

I've added `host` to my left prompt elements and it's working, but it has a terminal icon preceding it:

![image](https://github.com/user-attachments/assets/4fc72f6b-ecd0-477a-8b52-532de1eb6c2a)

Is there a way to hide that?

Thanks!

**Comment by romkatv** [comment]
`POWERLEVEL9K_HOST_VISUAL_IDENTIFIER_EXPANSION=`

**Reply by binaryfire** [reply]
Perfect, thanks!
